docs/concepts.md

```diff
@@ -215,17 +216,17 @@ Intuitively, data can only flow from `a` to `b` if:
 The least privileged label is usually referred to as "public trusted" (and
```

Comment:
Not related to the change you just made, but I would consider changing this to say:

"The information flow label  "public trusted" corresponds to a Node or Channel which has only observed public data, and is universally trusted. This is the bottom of the information flow label (and it may also be represented as `⊥`, pronounced "bottom"), which means information from sources with this label are allowed to influence anything."

Reasons for this suggestion:
- Avoids a non-standard use of "endorsed" -- endorsed usually means the integrity label has changed (down the lattice)
- Updates some prose around the explanation of the "trusted" part of this label (It sounds like it used to be "untrusted")
- [I am somewhat less convinced about this part of the change myself] avoids the word privilege, because maybe privilege implies "authority lattice" wherein public_untrusted has the least privilege (can't declassify anything, can't endorse anything). Though, I suppose the information flow lattice is also about privilege in a way because it is privilege to receive information in the first place.

Reply:
I have another PR that clarifies some of this, I'll incorporate your feedback in that!

Reply:
OK, SG approved.